test/Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -2035,8 +2094,6 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
 
             // All OnCompleted callbacks should be called even if they throw.
             Assert.Equal(2, testLogger.ApplicationErrorsLogged);
-            Assert.True(onCompletedCalled1);
-            Assert.True(onCompletedCalled2);
```

Comment:
Why remove these checks?

Reply:
If the OnCompleted handlers are called we should have exactly 2 exceptions. I guess those exceptions could be from elsewhere, so I'll add it back.